.github/workflows/bench.yml

```
@@ -0,0 +1,58 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  bench:
+    name: Performance Regression Tests (Baseline)
+    permissions:
+      checks: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bencherdev/bencher@main
+      - name: Use Rust Stable
+        uses: dtolnay/rust-toolchain@stable
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: npm
+      - name: npm install
+        run: npm ci --prefer-offline --no-audit --no-fund
+      - name: Install Benchmark Dependencies
+        with-directory: ./bench
```

```
The workflow is not valid. .github/workflows/bench.yml (Line: 28, Col: 9): Unexpected value 'with-directory' .github/workflows/bench.yml (Line: 31, Col: 9): Unexpected value 'with-directory'Show less
```

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/setting-a-default-shell-and-working-directory#example-set-the-default-shell-and-working-directory

```suggestion
        working-directory: ./bench
```